RUFAS/routines/animal/genetics/animal_genetics.py

```diff
@@ -1,15 +1,211 @@
+import numpy as np
+
+from RUFAS.input_manager import InputManager
 from RUFAS.time import Time
+from RUFAS.util import Utility
+
+import datetime
+
+base_change_lookup_table = {
+    datetime.date.fromisoformat("2020-04-01"): 231,
+    datetime.date.fromisoformat("2014-12-01"): 184,
+    datetime.date.fromisoformat("2010-01-01"): 132,
+    datetime.date.fromisoformat("2005-01-01"): 0,
+}
```

Comment:
Could you please add a d
```suggestion
BASE_CHANGE_LOOKUP_TABLE = {
    datetime.date.fromisoformat("2020-04-01"): 231,
    datetime.date.fromisoformat("2014-12-01"): 184,
    datetime.date.fromisoformat("2010-01-01"): 132,
    datetime.date.fromisoformat("2005-01-01"): 0,
}
```
Could you please add an explanation of what this table is and its source(s)?